### ISSUE TITLE
这里缺失了一段

### ISSUE THREAD

**Issue by 20170702102** [MatNLP/HiCLRE#1]
https://github.com/MatNLP/HiCLRE/blob/54e6beda51858d4dcc33d3d4266e50b606fa5f5b/hiclre/framework/data_loader.py#L221

**Comment by wangxinzhe123** [comment]
请问您解决了这个问题吗？

**Comment by qq31415926** [comment]
这一行代码应该是
`seqs[i] = torch.cat(seqs[i],dim=0)`

**Comment by fromdongyang** [comment]
> 这一行代码应该是 `seqs[i] = torch.cat(seqs[i],dim=0)`

正解，多谢。
今天突然看到，回复晚了，各位抱歉
